.github/workflows/main.yml

```
@@ -20,7 +20,7 @@ jobs:
       - name: Setup ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 3.4.7
+          ruby-version: 3.4.8
```

Why pin it specific z-streams and not just 3.4?
```suggestion
          ruby-version: 3.4
```